modules/apps/object/object-test/src/testFunctional/macros/CreateObject.macro

```diff
@@ -40,6 +40,12 @@ definition {
 			locator1 = "CreateObject#OBJECT_NAME");
 	}
 
+	macro assertCustomObjectNotPresent {
+		AssertTextNotEquals(
+			locator1 = "ObjectPortlet#SEARCH_RELATIONSHIP_ENTRY",
+			value1 = "${customObject}");
+	}
```

Comment:
Please, you can change the name of the macro for `assertEntryOnRelationshipFieldNotPresent` and change the value `customObject` for `entryValue`. 
Please, change the file of macro for `ObjectPortlet.macro`